lib/index.js

```diff
@@ -19,6 +20,12 @@ var KISSmetrics = module.exports = integration('KISSmetrics')
   .retries(2);
 
 /**
+ * Use a POST request if the GET query string payload exceeds 2KB.
+ */
+
+var MAX_GET_PAYLOAD = 1 << 11;
```

Comment:
Shifting is used to represent an exponent of 2, and 2 to the power of 10 is 1024 so I'd say 2<<10 is the most confusing. I've updated it to a literal in 2a7d6391fff3178e0cea796eb329f31a2f54e4ac.